## Ejercicio 1 - Utilizar operadores aritméticos

Python proporciona operadores aritméticos comunes para que puedas realizar operaciones matemáticas en tu código. Estos incluyen las cuatro operaciones principales de suma, resta, multiplicación y división.

Exploremos cómo podemos crear un programa que pueda calcular la distancia entre dos planetas. Comenzaremos usando dos distancias de planetas: Tierra (149.597.870 km) y Júpiter (778.547.200 km).

In [4]:
# Distancia de los planetas en km
tierra = 149597870
jupiter = 778547200

Realizar la operación

Con los valores obtenidos, es el momento de añadir el código para realizar la operación. Restarás el primer planeta del segundo para determinar la distancia en kilómetros. A continuación, puedes convertir la distancia del kilómetro en millas multiplicándola por 0.621.

In [17]:
diferencialkm = jupiter - tierra
diferencialmi = round(diferencialkm*0.621)
print("La distacncia entre planetas es de", diferencialkm, "km o", diferencialmi, "millas")

La distacncia entre planetas es de 628949330 km o 390577534 millas


Prueba tu proyecto

Con el código creado, ejecuta el notebook para obtener el resultado. Deberías recibir lo siguiente:

628949330

390577534

## Ejercicio 2: convierte cadenas en números y usa valores absolutos

#### Crear una aplicación para trabajar con números y entrada de usuario
Con frecuencia, deberás convertir los valores de cadena en números para realizar correctamente diferentes operaciones o determinar el valor absoluto de un número.

Para crear nuestra aplicación, queremos leer la distancia del sol para dos planetas, y luego mostrar la distancia entre los planetas. Haremos esto usando input para leer los valores, int para convertir a entero y luego abs para convertir el resultado en su valor absoluto.

#### Lee los valores
Usando input, agrega el código para leer la distancia del sol para cada planeta, considerando 2 planetas.

Planeta/Distancia al sol

Mercurio/57900000
Venus/108200000
Tierra/149600000
Marte/227900000
Júpiter/778600000
Saturno/1433500000
Urano/2872500000
Neptuno/4495100000

In [26]:
planeta1 = input("Ingrese la distancia del primer planeta al Sol en km")
planeta2 = input("Ingrese la distancia del segundo planeta al Sol en km")

#### Convertir a número
Debido a que input devuelve valores de cadena, necesitamos convertirlos en números. Para nuestro ejemplo, usaremos int

In [27]:
planeta1 = int(planeta1)
planeta2 = int(planeta2)

#### Realizar el cálculo y convertir a valor absoluto
Con los valores almacenados como números, ahora puedes agregar el código para realizar el cálculo, restando el primer planeta del segundo. Debido a que el segundo planeta podría ser un número mayor, usarás abs para convertirlo a un valor absoluto. También agregarás el código para mostrar el resultado en millas multiplicando la distancia del kilómetro por 0.621

In [29]:
distEntrePlanetasKm = abs(planeta2 - planeta1)
distEntrePlanetasMi = distEntrePlanetasKm*0.621
print("La distancia entre planetas es de", distEntrePlanetasKm, "km o", distEntrePlanetasMi, "millas.")

La distancia entre planetas es de 4437200000 km o 2755501200.0 millas.
